phi/assistant/assistant.py

```diff
@@ -218,62 +209,8 @@ def set_run_id(cls, v: Optional[str]) -> str:
     def streamable(self) -> bool:
         return self.output_model is None
```

Comment:
Maybe with llama-index integration, we can stream partial intermediate pydantic object?

https://docs.llamaindex.ai/en/stable/examples/output_parsing/openai_pydantic_program/?h=stream+program#stream-partial-intermediate-pydantic-objects

Reply:
@sandangel this is on the list for us to implement :) maybe i hack on it tonight :)